app/api/chemotion/attachment_api.rb

```diff
@@ -100,7 +100,7 @@ def upload_chunk_error_message
       post 'upload_dataset_attachments' do
         error_messages = []
         params.each do |_file_id, file|
-          next unless tempfile = file[:tempfile]
+          next unless tempfile = file[:tempfile] # rubocop:disable Lint/AssignmentInCondition
```

Comment:
I do not really know. It seems to be unused because it is only used by AttachmentFetcher.js in method uploadFiles (line 187), which seems not to be called itself. 

Can somebody deeper in that area of code confirm this?

Reply:
probably could have been removed  with that 6cf21909d7

Reply:
will be removed in the next step then while cleaning lib/storage